src/components/signin/components/Signin.js

```diff
@@ -49,7 +49,10 @@ class Signin extends React.Component {
 
     render() {
         const { signinMessage, signinStatus } = this.props;
-
+        const { resetSignState } = this.props.actions;
+        const notifySigninStatus =() => signinStatus ? message.success(signinMessage, 5) : 
+            message.error(signinMessage, 5);
+        resetSignState();
```

Comment:
You are calling `resetSignState` every time on rerender this is wrong. Only call after you have shown the message